src/lib-components/complex-legend.vue

```diff
@@ -191,7 +191,7 @@ export default defineComponent({
     fill: rgba(0, 0, 0, 0.3);
   }
   & .phase-arc {
-    stroke: rgba(0, 0, 0, 0.1);
+    stroke: rgba(0, 0, 0, 0.5);
```

Comment:
I preffered the previous option

Reply:
The previous option doesn't work well with the oklab implementaion.

Previous option
<img width="686" alt="16" src="https://user-images.githubusercontent.com/16420546/182667483-40fb6e58-b436-48c7-9ff1-0592a61d744f.png">

Current option
<img width="685" alt="17" src="https://user-images.githubusercontent.com/16420546/182667664-5294e3d0-3e06-4c04-a108-dbc87ab70f7a.png">